# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: accessible/generic/LocalAccessible.cpp
# Commit: 92f4ad55d56d
# Full Hash: 92f4ad55d56d451307f2c00fc7657b64266d71b8
# Author: James Teh <jteh@mozilla.com>
# Date: 2021-11-22 15:43:20
# Description:
#   Bug 1742003: Handle mContent being null when caching DOM node id. r=eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D131732
# ==============================================================================

diff -r ace2f4af2c29 -r 92f4ad55d56d accessible/generic/LocalAccessible.cpp
--- a/accessible/generic/LocalAccessible.cpp	Mon Nov 22 05:34:14 2021 +0000
+++ b/accessible/generic/LocalAccessible.cpp	Mon Nov 22 05:39:39 2021 +0000
@@ -3242,8 +3242,7 @@
     }
   }
 
-  if (aCacheDomain & CacheDomain::DOMNodeID) {
-    MOZ_ASSERT(mContent);
+  if (aCacheDomain & CacheDomain::DOMNodeID && mContent) {
     nsAtom* id = mContent->GetID();
     if (id) {
       fields->SetAttribute(nsGkAtoms::id, id);
